lib/Sema/CSDiagnostics.h

```diff
@@ -1075,6 +1075,14 @@ class MissingMemberFailure : public InvalidMemberRefFailure {
   /// that defaults the element type. e.g. _ = [.e]
   bool diagnoseInLiteralCollectionContext() const;
 
+  /// Tailored diagnostics for missing subscript member on a tuple base type.
+  /// e.g
+  /// ```swift
+  ///   let tuple: (Int, Int) = (0, 0)
+  ///   _ = tuple[0]
```

Comment:
Nit: Could you replace `_ = tuple[0]` with `_ = tuple[0] // -> tuple.0`.

Reply:
Sure :)